docs/source/documentation/team/wow-2.html.md.erb

```diff
@@ -0,0 +1,222 @@
+
+### Sprint Review
+
+- Review completed work with stakeholders, gathering feedback for future sprints. Usually encorporated with Planning.
+
+#### Audience
+
+DPAT team
+
+### Sprint Retrospective
+
+- Reflect on the sprint’s process, celebrating successes and identifying opportunities for improvement
+- Review the process, not the stories
+- Define how we could improve
+- Share kudos and feedback
+
+#### Audience
+
+DPAT team
+
+### Show and Tell
+
+- At the end of each sprint we have a thirty-minute show and tell showcase to share with our colleagues and the wider organisation our achievements during the sprint. At the end of each sprint we have a thirty-minute show and tell showcase to share with our colleagues and the wider organisation our achievements during the sprint. The showcase normally consists of an introduction summing up the sprint and if we met our sprint goal followed by demonstration and then what’s next in the road map
+
+#### Audience
+
+All of Data Platform and wider audience
+
+## GitHub Practices
+
+- We utilise GitHub for managing code, documentation, and support, with GitHub Projects for tracking work and GitHub pull requests for reviewing and merging code
+- When working in our repos, please adhere to GitHub housekeeping guidelines when creating pull requests, ensuring clear titles and descriptions, linking to issues when applicable, and requesting reviews upon readiness
+
+### GitHub Housekeeping
+
+- When creating a pull request, ensure you have a clear title and description
+- If applicable, link your pull request to an issue with the special keywords such as `closes`, `fixes` or `resolves` [GitHub documentation for linking pull requests to issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
+- If your pull request is not ready for review, convert it to a draft
+- When your pull request is ready for review, remove the draft status and ensure all checks are passing
+- When all checks are passing, use the `/pull` command in [`#ask-data-platform`](https://mojdt.slack.com/archives/C04VBJTKY58) to request a review
+
+## Development Practices
+
+- This is quite a large topic, so we maintain a separate page for it: [Developing](/documentation/platform/infrastructure/developing.html)
+
+## Support
+
+- Each member of the team is included in the support rota
+- Support cover is for one day (10:00-16:30) as allocated by the rota
+- If you are unable to cover support please arrange cover
+- PagerDuty is used to manager our support rota link - https://moj-digital-tools.pagerduty.com/schedules-new
+
+## Support Duties
+
+- Monitoring the slack support channel  https://moj.enterprise.slack.com/archives/C4PF7QAJZ
+- Actioning raised issues or seek support from team to do so
+- Run that days standup
+- Respond to PagerDuty alerts, [support issues on GitHub](https://github.com/ministryofjustice/data-platform-support/issues), and review pull requests as part of support duties
+
```

Comment:
I think we should document some general practices, such as using labels?

```suggestion
In order to keep the backlog of support issues tidy and keep our users updated, when on support follow these best practices:
- When a new issue is raised in GitHub remove the `triage` label and assign the issue to yourself
- Add the `investigating` label to indicate to the user that you are looking into the issue
- Add the `awaiting-response` label if you reply to the user to ask a question or request further information
- Close the issue in GitHub when it is resolved, documenting the steps you took to resolve it, and any further relevant information e.g. link to team discussions in slack. This will help future support leads find common solutions and related issues

```

Reply:
💯 if we could add "make sure the labels are correct as they help with understanding support is being consumed"

Reply:
Updated my suggestion to include that